src/main/kotlin/money/Amount.kt

```diff
@@ -0,0 +1,21 @@
+package money
+
+import java.lang.IllegalArgumentException
+
+class Amount(private val stringAmount: String) {
+    private val integers = '0'..'9'
+
+    init {
+        if (stringAmount.isBlank()) {
+            throw IllegalArgumentException("값이 비어있습니다.")
+        }
+
+        if (!isInteger(stringAmount)) {
+            throw IllegalArgumentException("입력한 $stringAmount 값은 숫자여야 합니다.")
+        }
+    }
+
+    fun getBigDecimal() = stringAmount.toBigDecimal()
+
+    private fun isInteger(input: String) = input.all { it in integers }
```

Comment:
```suggestion
    private fun String.isInt() = this.all { it in integers }
```

코틀린에는 [Extension function(확장 함수)](https://kotlinlang.org/docs/extensions.html) 라는 재미있는 기능이 있는데요, Java 에서는 String 클래스를 개발자가 제어할 수 없었지만 코틀린에서는 이렇게 함수를 추가하는 것도 가능합니다. 이렇게 확장함수를 정의하면,

```kotlin
require(stringAmount.isInt()) { "입력한 $stringAmount 값은 숫자여야 합니다." }
```

이렇게 축약하는 것도 가능합니다!

코틀린에는 Integer 타입이 없기 때문에 메서드 이름을 `isInt()` 로 쓰시는건 어떨까 싶네요~

Reply:
`isInt` 좋습니다!